documentation/src/docs/python.md

```diff
@@ -0,0 +1,130 @@
+# Python API for the connector
+
+This Pravega connector of Python API provides a data source and data sink for Flink streaming jobs.
+
+Your Flink streaming jobs could use Pravega as their storage with these [Python API Wrappers](https://github.com/pravega/flink-connectors/tree/master/src/main/python).
+
+**DISCLAIMER: This python wrapper is an IMPLEMENTATION REFERENCE and is not meant for out-of-box usage.**
+
+[TOC]
+
+## How to use
+
+Together with python wrapper files, you could submit your job with main compute code like this:
+
+```bash
+flink run --python ./application.py --pyFiles ./pravega_config.py --pyFiles ./pravega_writer.py --pyFiles ./pravega_reader.py --jarfile /path/to/pravega-connectors-flink.jar
+```
+
+## PravegaConfig
+
+A top-level config object, `PravegaConfig`, is provided to establish a Pravega context for the Flink connector.
+
+```python
+from pravega_config import PravegaConfig
+
+pravega_config = PravegaConfig(CONTROLLER_URI, SCOPE)
+```
+
+|parameter|type|required|default value|description|
+|-|-|-|-|-|
+|uri|str|Yes|N/A|The Pravega controller RPC URI.|
+|scope|str|Yes|N/A|The self-defined Pravega scope.|
+|schema_registry_uri|str|No|None|The Pravega schema registry URI.|
```

Comment:
We can skip the `schema_registry_uri` here because it's still not used until we support the deserialization with schema registry.